drivers/wifi/infineon/cybsp.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <cy_result.h>
+#include <cyhal_sdio.h>
+#include <zephyr/device.h>
+#include <zephyr/dt-bindings/pinctrl/ifx_cat1-pinctrl.h>
+
+/* WIFI interface types */
+#define CYBSP_SDIO_INTERFACE             (0)
+
+
+#define CYBSP_WIFI_INTERFACE_TYPE        CYBSP_SDIO_INTERFACE
+
+/**
+ * \brief Get the initialized sdio object used for communicating with the WiFi Chip.
+ * \returns The initialized sdio object.
+ */
+cyhal_sdio_t *cybsp_get_wifi_sdio_obj(void);
+
+
+/* WiFi host-wake IRQ event */
+#define CYBSP_WIFI_HOST_WAKE_IRQ_EVENT  (CYHAL_GPIO_IRQ_RISE)
+
+/* WiFi host-wake IRQ event */
+#define DT_WIFI_SDIO_NODE               DT_COMPAT_GET_ANY_STATUS_OKAY(infineon_cyw43xxx_wifi_sdio)
```

Comment:
Thanks for catching this in an earlier review, @gmarull -- yes, the `DT_` namespace in zephyr is reserved for macros defined in devicetree APIs: that is, `devicetree.h`, its sub-headers, and `include/dt-bindings/`. Please do not use `DT_` in names that are outside of these places.